src/test/java/org/fundacionjala/coding/german/SortInnerConntentTest.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.german;
+
+import org.junit.Test;
+
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by German on 8/29/2017.
+ */
+public class SortInnerConntentTest {
+    /**
+     * Verify  sortInnerConntent method.
+     */
+    @Test
+    public void testSortInnerConntentMethod() {
+        SortInnerContent sortInnerContent = new SortInnerContent();
```

Comment:
You are repeating this code in all your @Test methods.
Please move it to a @Before method.